uber/tests/models/test_merch.py

```diff
@@ -23,23 +23,23 @@ def donation_tier_fixture(monkeypatch):
 
 
 class TestMerchAttrs:
```

Comment:
This is going to look bigger than it really is because of the rename of "swag" to "event".

Other than that, I added new tests for all of the new properties on the ``Attendee`` class, e.g. ``num_staff_shirts_owed`` and ``shirt_info_marked``, etc.